addons/account/models/account_move.py

```diff
@@ -557,6 +557,9 @@ def _auto_init(self):
             ON account_move(journal_id) WHERE to_check = true;
             CREATE INDEX IF NOT EXISTS account_move_payment_idx
             ON account_move(journal_id, state, payment_state, move_type, date);
+            -- Used for gap detection in list views
+            CREATE INDEX IF NOT EXISTS account_move_sequence_index3
```

Comment:
Is there a reason for a '3' at the end of the name there?

Reply:
It's because 1 and 2 are created in the `sequence.mixin`, but the gap detection is only on `account.move`